routes/router.ts

```diff
@@ -1,13 +1,45 @@
 import express, { Request, Response } from 'express';
-import { getAllCategoriesController } from '../controllers/categoriesController';
-
+import { getAllCategoriesController, carouselImagesController } from '../controllers/categoriesController';
```

Comment:
This better to be named as `CategoryController` and in the routers you can can use it like `CategoryController.getAllCategories` (rename the function as well).